src/foam/nanos/dig/SUGAR.js

```diff
@@ -41,205 +39,26 @@ foam.CLASS({
       name: 'id',
       displayWidth: 40
     },
-    {
-      class: 'String',
-      name: 'serviceKey',
```

Comment:
Why removing the serviceKey? How will the user know what services are available?

Reply:
this doesn't work, that's why I'm removing it.

Reply:
They will have to find out what is available through documentation

Reply:
I would rather just fix it, since the dynamically generated list which is based on their current permissions and capabilities will always be correct, whereas documentation may not be.